examples/src/main/java/com/mapbox/navigation/examples/core/SimpleMapboxNavigationKt.kt

```diff
@@ -113,6 +114,7 @@ class SimpleMapboxNavigationKt : AppCompatActivity(), OnMapReadyCallback {
         val newOptions =
             options.toBuilder()
                 .onboardRouterConfig(onboardRouterConfig)
+                .fasterRouteDetectorInterval(TimeUnit.MINUTES.toMillis(1))
```

Comment:
Am I reading correctly that this will ping the Directions API every minute for a new route?

Reply:
What request options are included in the API call for a faster route? You might want to consider including the driver's current bearing if they're moving at-speed, this will help ensure a faster route behind them is not selected.

Reply:
@danpaz yeah the default is 2 minutes and we give that knob to customers (I made it 1 for testing). do you have a recommended interval?  

good point on the bearing, I think we are doing that but that's a different part of the code

Reply:
> Am I reading correctly that this will ping the Directions API every minute for a new route?

That's correct, in any case that's customizable and in that example is set to 1 minute.

Ohh @kmadsen beat me to answer :rocket:

Reply:
> do you have a recommended interval?

Based on how frequently traffic changes and our speed data refreshes, I'd default to something closer to 5-10 minutes. Hard to say without user testing whether this frequency and the 90% threshold strikes the right balance of annoying vs. time saving.

Reply:
for what it's worth we had similar issues to this at lyft. evolution went like this
1. part of a location update (decided from a every 5 seconds interval)
2. separate endpoints (every x seconds/minutes)
3. server controlled configuration (polling rate decided by the server in response headers)
4. push to client (no interval on client)

i'm open to the slower interval. but am leaning towards faster is better because it's we can still drop requests?

Reply:
The reason I suggest using as large a refresh interval as possible is that these Directions API calls are not free. The faster route feature will have a multiplier effect on the number of API calls a navigation app would make during a trip (potentially hundreds more API calls per trip than before this feature for a longer trip if I understand correctly). 

This could represent significant additional load on our API servers, though I'm less concerned about that. From a user's perspective this would represent additional bandwidth, battery, etc., and more billable requests, depending on how they're billed. The server-driven approach you laid out sounds more robust for sure but we're pretty far from supporting that on the Directions API. /cc @mapbox/navigation-api